uint32 trayId
---
TrayState state

